File: apiSendMenssage.php
<?php

// Defina o token de autenticação
$token = '';

$data = [
    'number' => '5571982462438', // Número de telefone no formato internacional
    'body' => 'Ei Rony, esta é uma mensagem enviada apenas como texto pelo PHP!',
    'externalKey' => ''//Esta chave é a ultima parte dos caracteres da url!,logo se a url for: "https://api.zip-zop.site/v1/api/external/asdl-asdmasd=3323d" a externalKey será:"asdl-asdmasd=3323d"
];

// URL do endpoint
$url = '';


// Inicializar cURL
$ch = curl_init($url);

// Configurar os headers e a requisição
curl_setopt($ch, CURLOPT_HTTPHEADER, [
    'Authorization: Bearer ' . $token,
    'Content-Type: application/json'
]);
curl_setopt($ch, CURLOPT_POST, 1);
curl_setopt($ch, CURLOPT_POSTFIELDS, json_encode($data));
curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);

// Adicionar verbosidade para debug
curl_setopt($ch, CURLOPT_VERBOSE, true); // Ativar verbosidade
$verboseLog = fopen('php://temp', 'w+'); // Criar log temporário
curl_setopt($ch, CURLOPT_STDERR, $verboseLog);

// Executar a requisição
$response = curl_exec($ch);

// Verificar se ocorreu algum erro
if (curl_errno($ch)) {
    echo 'Erro: ' . curl_error($ch);
} else {
    // Mostrar a resposta da API
    echo "Resposta da API:\n" . $response . "\n";
}

// Capturar e exibir o log de debug
rewind($verboseLog);
$verboseOutput = stream_get_contents($verboseLog);
fclose($verboseLog);

echo "\nLog de Debug:\n" . $verboseOutput;

// Fechar a conexão cURL
curl_close($ch);

File: apiSendMenssage.py
import requests

# Mensagem e número do destinatário
body = "Ei Zip-Zop, esta é uma mensagem enviada apenas como texto pelo Python!"
number = "557183196364"

# URL do endpoint e token
url = 'URL_API'
token = "TOKEN_API"

# Dados do formulário (apenas texto)
data = {
    "number": number,
    "body": body,
}

# Cabeçalhos da requisição
headers = {
    "Authorization": f"Bearer {token}",
    "Content-Type": "application/json",  # Enviando os dados no formato JSON
}

# Enviar a requisição
try:
    response = requests.post(url, json=data, headers=headers)

    # Verifica se houve erro na requisição
    if response.status_code != 200:
        print(f"Erro na requisição: {response.status_code} - {response.reason}")
        print("Detalhes:", response.text)
        exit(1)

    # Exibe a resposta da API
    print("Resposta da API:")
    print(response.text)

except requests.RequestException as e:
    print("Erro ao enviar a mensagem:", e)
